frontend/src/global_styles/layout/work_packages/_table.sass

```diff
@@ -71,7 +71,14 @@ body[class*="router--"]
   div.row-hovered
     // Use important to override the background styles from __hl classes
     // That also have to be important (cf. 30863)
-    background: var(--bgColor-muted) !important
+    background: var(--bgColor-neutral-muted) !important
+
+    td
+      border-top: 1px solid var(--borderColor-neutral-muted) !important
```

Comment:
It looks weird, that the border is thus not visible any more:
<img width="1065" alt="Bildschirmfoto 2024-08-12 um 13 03 03" src="https://github.com/user-attachments/assets/749e9265-fee1-4533-822b-255b9a13a9fa">

**Before**
<img width="1090" alt="Bildschirmfoto 2024-08-12 um 13 03 25" src="https://github.com/user-attachments/assets/0506ba1b-cec4-400b-9f74-6de27dfc9c34">